redirects.yaml

```diff
@@ -1,3 +1,27 @@
+# Wordpress CMS pages & resources
 /admin(?P<page>/.*)?: https://admin.insights.ubuntu.com/admin{page}
-/feed(?P<page>/.*)?: https://admin.insights.ubuntu.com/feed{page}
 /wp-(?P<page>.*)?: https://admin.insights.ubuntu.com/wp-{page}
+
+# Feeds
+/feed/?: https://admin.insights.ubuntu.com/feed
+/(?P<page>.*)/feed/?: https://admin.insights.ubuntu.com/{page}/feed
+
+# Archive pages
+/page/(?P<page>[0-9]+)/?: /archives?page={page}
+
+# Category archive pages
+/articles/?: /archives?category=articles
```

Comment:
I didn't realise all categories were archives now. Interesting.

Reply:
You can access them through a filter on anything:

Homepage: `/?category=articles`
Group page: `/cloud-and-server?category=articles`
Archives: `/archives?category=articles&group=canonical-announcements`